MonoGame.Framework/Color.cs

```diff
@@ -216,9 +216,9 @@ public Color(Color color, int alpha)
             B = color.B;
             A = (byte)MathHelper.Clamp(alpha, Byte.MinValue, Byte.MaxValue);
         }
-	
-	/// <summary>
-        /// Creates a new instance of <see cref="Color"/> struct.
+
+        /// <summary>
+        /// Constructs an RGBA color from color and alpha value(0.0-1.0).

```

Comment:
Instead of putting these in parenthesis... just make it say this in the description:
```
Constructs an RGBA color from a Color and an alpha value.
```
Then in the description for the `alpha` parameter do:
```
The alpha value which will be clamped 0 to 1.
```
It would be better to do something like this for the other cases below.